calendario/migrations/0008_auto_20191116_1616.py

```diff
@@ -12,6 +12,6 @@ class Migration(migrations.Migration):
     operations = [
         migrations.AlterModelOptions(
             name='turno',
-            options={'permissions': [('can_schedule_turno', 'Puede agendar un turno'), ('can_view_misturnos', 'Puede ver Mis Turnos'), ('can_cancel_turno', 'Puede cancelar sus turnos'), ('can_gestionar_turnos', 'Puede gestionar turnos')]},
+            options={'permissions': [('can_schedule_turno', 'Puede agendar un turno'), ('can_view_misturnos', 'Puede ver Mis Turnos'), ('can_cancel_turno', 'Puede cancelar sus turnos')]},
```

Comment:
En general esta mal reescribir a mano la historia de los migrates de Django. Es peligro y puede terminar mal.
Lo mejor es cambiar el modelo y dejar que django vuelva atras con las cosas

```suggestion
            options={'permissions': [('can_schedule_turno', 'Puede agendar un turno'), ('can_view_misturnos', 'Puede ver Mis Turnos'), ('can_cancel_turno', 'Puede cancelar sus turnos'), ('can_gestionar_turnos', 'Puede gestionar turnos')]},
```

Reply:
Coincido con lo peligroso de manipular las migraciones.
En este caso la modifiqué porque la migración 0008 agregó el permiso `can_gestionar_turnos`, y la 0009, que sería la que omití, lo quitaría.

Ya agrego entonces la nueva migración.